modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFormInstanceRecordVersionLocalServiceImpl.java

```diff
@@ -77,6 +77,14 @@ public DDMFormInstanceRecordVersion getFormInstanceRecordVersion(
 				ddmFormInstanceRecordId, start, end, orderByComparator);
 	}
 
+	@Override
+	public List<DDMFormInstanceRecordVersion> getFormInstanceRecordVersions(
+		long userId, long formInstanceId) {
```

Comment:
@marcelabc,
Although `ddmFormInstanceId` is used in other methods, `formInstanceId` makes more sense to me. So let's keep it the way you did 😄.